core-bundle/tests/Fixtures/vendor/contao/test-bundle/Resources/contao/languages/en/default.xlf

```diff
@@ -21,8 +21,10 @@
                 <target>This is the fourth target</target>
             </trans-unit>
             <trans-unit id="MSC.fifth">
-                <source>This is the\nfifth source</source>
-                <target>This is the\nfifth target</target>
+                <source>This is the
+fifth source</source>
+                <target>This is the
+fifth target</target>
```

Comment:
If `\n` no longer works, that is a BC break, isn‘t it?

Reply:
The handling was different between `dev` and `prod`, so it did not work properly before anyway.

Reply:
How does the Symfony translator handle this case?

Reply:
In YAML files you can use `|` in order to create translation labels with line breaks: https://symfony.com/doc/current/reference/formats/yaml.html#strings

Reply:
That‘s great, but it does not answer the question. If the Symfony translator converts a literal `\n` in an .xliff file to a real line-break, the change is a BC break. However, if it is only a "feature" of our cache warmer, we can drop it.

Reply:
The Symfony translator does not convert `\n` or anything else inside `<target>` (it only does a UTF-8 conversion if the .xlf file is not stored as UTF-8).